libs/common-const/src/tokens.ts

```diff
@@ -23,14 +23,14 @@ export const USDC_MAINNET = new Token(
   '0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48',
   6,
   'USDC',
-  'USD//C'
+  'USD Coin'
 )
 export const USDC_GOERLI = new Token(
   SupportedChainId.GOERLI,
   '0x07865c6e87b9f70255377e024ace6630c1eaa37f',
   6,
   'USDC',
-  'USD//C'
+  'USD Coin'
```

Comment:
This works on a fresh page, doesn't seem to if the website was used before...

![image](https://github.com/cowprotocol/cowswap/assets/43217/ebcbf1ab-3335-4867-b669-fbf336ac114e)

I'm guessing this is because of cached token lists.

Reply:
Actually, the bad cached name is on my favourite tokens 
![image](https://github.com/cowprotocol/cowswap/assets/43217/060112aa-94e4-4d65-90ad-6257986292ad)

Reply:
Resolved hackishily by mapping token name `USD//C` to `USD Coin` 😬